client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -280,21 +276,20 @@ class OutlinePlugin: CDVPlugin {
     // MARK: Callback helpers
 
     private func sendSuccess(callbackId: String, keepCallback: Bool = false) {
+        DDLogInfo("CordovaPlugin Success")
```

Comment:
I don't think this is meaningful for Info logs. Make it Debug. Perhaps add more info.